src/arg_parse.rs

```diff
@@ -585,9 +585,9 @@ EXAMPLE USE:
         is_terminal: atty::is(Stream::Stdout),
         no_color: args.is_present("no_color"),
         disable_validator: args.is_present("disable_validator"),
-        http_verb: value_t!(args.value_of("http_verb"), HttpVerb).unwrap()
+        http_verb: value_t!(args.value_of("http_verb"), HttpVerb).unwrap(),
         response_min_length: args.value_of("response_min_length").unwrap().parse::<u32>().unwrap(),
-        response_max_length: args.value_of("response_max_length").unwrap().parse::<u32>().unwrap()
+        response_max_length: args.value_of("response_max_length").unwrap().parse::<u32>().unwrap_or(std::u32::MAX)
```

Comment:
Potential issue with this line in that user-supplied invalid values will be silently converted to int_max, maybe it would be better to write a dedicated validation function.